ebpf.go

```diff
@@ -91,7 +92,8 @@ func (e ebpfOpts) label(name string) string {
 }
 
 func (e ebpfOpts) stackOffset(n int) int16 {
```

Comment:
So `n` is the slot?

Reply:
Yeah. Naming around this is hard, the BPF spec just calls it `m[]`, and the `x/net` library uses `N` (https://godoc.org/golang.org/x/net/bpf#LoadScratch). Renamed to `scratch`, and added a comment :)